src/Injector.ts

```diff
@@ -48,9 +68,9 @@ export function Injector<C, T extends Constructor<BaseInjector<C>>>(Base: T, con
 			const {
 				render,
 				properties,
-				getProperties,
+				getProperties = defaultMappers.getProperties,
```

Comment:
Roger that, kinda of a change outside the scope of this PR so missed it. The change is needed tho, this is the first real usage of the injector pattern (so found some small fixes were required.)